controllers/controller.js

```diff
@@ -20,3 +25,23 @@ exports.getArticleById = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.patchArticlebyId = async (req, res, next) => {
+  try {
+    const article_id = req.params.article_id;
+    const votes = req.body.inc_votes;
+    const patcher = await patchArticleUsingId(article_id, votes);
```

Comment:
I'd suggest changing the variable name 'patcher'; it implies that it patches, which is better suited to a function rather than a returned variable value